packages/experimental/src/experimental-list/task-item.tsx

```diff
@@ -72,6 +73,27 @@ const OptionalTaskTooltip: React.FC< {
 	return <Tooltip text={ tooltip }>{ children }</Tooltip>;
 };
 
+const OptionalExpansionWrapper: React.FC< {
+	expandable: boolean;
+	expanded: boolean;
+} > = ( { children, expandable, expanded } ) => {
+	if ( ! expandable ) {
+		return <>{ children }</>;
+	}
+	return (
```

Comment:
This way we don't use the VerticalCSSTransition component when `expandable` is disabled.